test/tests/functional/pbs_ralter.py

```diff
@@ -391,11 +390,24 @@ def alter_a_reservation(self, r, start, end, shift,
                 msg = "Resv;" + r + ";Reservation alter confirmed"
             else:
                 msg = "Resv;" + r + ";Reservation alter denied"
-
-            lines = self.server.log_match(msg, n='ALL', allmatch=True,
-                                          interval=2, max_attempts=30)
-            self.assertEqual(len(lines), sequence)
-
+            attempts = 1
+            interval = 0.5
+            max_attempts = 30
+            while attempts <= max_attempts:
```

Comment:
You are iterating.  Use for attempts in range (1, max_attempts+1):

Reply:
Modified